OrchestratorIntegration.md

```diff
@@ -0,0 +1,51 @@
+Canal provides two things
+
+- Networking between hosts in your cluster
+- Enforcement of network policy
+
+Both areas require different choices for orchestrator integrators and are covered in detail below.
```

Comment:
I think this doc could use with a "this document does X" type statement, like the install guide.

"This document discusses the decisions that must be made when integrating canal with a new orchestrator" or something similar.